package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "updated-object-diff",
+  "version": "1.0.0",
+  "description": "A small library that returns the difference between two JavaScript objects, including nested objects, that handles values of all data types.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "prettier": "./node_modules/.bin/prettier . --write"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lyst/updated-object-diff.git"
+  },
+  "keywords": [
+    "updated",
+    "object",
+    "obj",
+    "diff",
+    "difference",
+    "deep",
+    "nested"
+  ],
+  "author": "Lyst",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/lyst/updated-object-diff/issues"
+  },
+  "homepage": "https://github.com/lyst/updated-object-diff#readme",
+  "dependencies": {
+    "jest": "^26.6.3",
```

Comment:
We could put jest & prettier in `devDepencies` to indicate they are only required for local development & testing.. [More info here. ](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file)